notebooks/236-raft-optimization/236-raft-optimization.ipynb

```diff
@@ -0,0 +1,570 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Convert and Optimize RAFT with OpenVINO™\n",
+    "\n",
+    "The Recurrent All-Pairs Field Transforms (RAFT) [repository](https://github.com/princeton-vl/RAFT) is a new deep network architecture for optical flow. \n",
+    "\n",
+    "RAFT extracts perpixel features, builds multi-scale 4D correlation volumes for all pairs of pixels, and iteratively updates a flow field through a recurrent unit that performs lookups on the correlation volumes. RAFT achieves stateof-the-art performance.\n",
+    "\n",
+    "This tutorial demonstrates step-by-step instructions on how to run and optimize RAFT with OpenVINO based on the [ONNX-RAFT-Optical-Flow-Estimation](https://github.com/ibaiGorordo/ONNX-RAFT-Optical-Flow-Estimation/tree/main). \n",
+    "\n",
+    "The tutorial consists of the following steps:\n",
+    "- Download ONNX model\n",
+    "- Convert ONNX model to OpenVINO IR\n",
+    "- Prepare and run optimization pipeline\n",
```

Comment:
I do not see running any optimization for model, is it in your plans to add in next commits?

Reply:
I could add PPP optimization, if needed.
Is it possible to quantize with NNCF or POT?

Reply:
if it is applicable, i think it can be added